project/hub.py

```diff
@@ -241,9 +248,9 @@ def get_bucket_list(self) -> ResourceList[Bucket]:
         r.raise_for_status()
         return ResourceList[Bucket](**r.json())
 
-    def get_bucket_by_id_or_name(self, bucket_id_or_name: str | UUID) -> Bucket | None:
```

Comment:
Any reason why this was renamed? From my end I can still get buckets by either their ID or name just fine.

Reply:
because a bucket should only be requested by its unique id and no longer by its name